src/pages/settings/Profile/ProfilePage.js

```diff
@@ -241,17 +254,36 @@ class ProfilePage extends Component {
                         <Text style={[styles.mt6, styles.mb6]}>
                             {this.props.translate('profilePage.tellUsAboutYourself')}
                         </Text>
-                        <FullNameInputRow
-                            firstName={this.state.firstName}
-                            firstNameError={PersonalDetails.getMaxCharacterError(this.state.hasFirstNameError)}
-                            lastName={this.state.lastName}
-                            lastNameError={PersonalDetails.getMaxCharacterError(this.state.hasLastNameError)}
-                            onChangeFirstName={firstName => this.setState({firstName})}
-                            onChangeLastName={lastName => this.setState({lastName})}
-                            style={[styles.mt4, styles.mb4]}
-                        />
+
+                        <View style={[styles.flexRow, styles.mt4, styles.mb4]}>
+                            <View style={styles.flex1}>
+                                <TextInput
+                                    inputID="firstName"
+                                    name="fname"
+                                    label={this.props.translate('common.firstName')}
+                                    value={this.state.firstName}
+                                    hasError={this.state.hasFirstNameError}
+                                    errorText={PersonalDetails.getMaxCharacterError(this.state.hasFirstNameError)}
+                                    onChangeText={firstName => this.setState({firstName})}
```

Comment:
None of these props should be passed to inputs in Form. The inputs will get it automatically from Form.